app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> {order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
+  validates :content, presence: true, length: {maximum:
+    Settings.micropost_max_length}
+  validate :picture_size
+
+  private
+
+  def picture_size
+    if picture.size > Settings.max_picture_size.megabytes
+      errors.add :picture, "should be less than 5MB"
```

Comment:
`"should be less than 5MB"`: Use I18n, please

Reply:
@VuThiTranVan I cannot use i18n in model.